apps/passport-server/src/services/generic-issuance/genericIssuanceService.ts

```diff
@@ -229,6 +242,44 @@ export class GenericIssuanceService {
   public async getAllPipelines(): Promise<Pipeline[]> {
     return this.pipelines;
   }
+
```

Comment:
fwiw it's not strictly necessary for *all* code related to generic issuance to live in this service